codersdk/deployment.go

```diff
@@ -444,6 +444,10 @@ var PostgresAuthDrivers = []string{
 	string(PostgresAuthAWSIAMRDS),
 }
 
+// PostgresConnMaxIdleAuto is the value for auto-computing max idle connections
+// based on max open connections.
+const PostgresConnMaxIdleAuto = "auto"
```

Comment:
Would it make sense to use `-1` or 0 instead for 'auto' here? That way, we let `coder/serpent` handle the type conversion.

Reply:
I think `"auto"` is more idiomatic; `-1` doesn't have any real meaning at face value.